core/libp2p/dht/dht_strategy.hpp

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_DHT_STRATEGY_HPP
+#define KAGOME_DHT_STRATEGY_HPP
+
+namespace libp2p::dht {
+  /**
+   * Distributed Hash Table
+   */
+  struct DHTAdaptor {
```

Comment:
It is supposed to be base for a class implementing some logic, not only state. I suppose class is more appropriate.

Reply:
struct/class thing is not only for state/etc.
If I used class, I would have written `public:` every time.